packages/python/plotly/_plotly_utils/tests/validators/test_pandas_series_input.py

```diff
@@ -40,7 +40,7 @@ def color_validator(request):
         "uint16",
         "uint32",
         "uint64",
-        "float16",
+        # "float16",
```

Comment:
@alexcjohnson Python 2.0 forbids creating float16 indxes:

![image](https://user-images.githubusercontent.com/20811121/225617670-e72f42df-f05b-46a2-9c33-a5716f3cb44c.png)

The easiest thing to do would be to drop this line here.